.github/action/create-env.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+env_keys = list(dict(os.environ).keys())
+
+out_file = ""
+
+for key in env_keys:
+    if key.startswith("ENVKEY_"):
+        out_file += key.split("ENVKEY_")[1] + "=" + os.environ.get(key) + "\n"
+
+with open( str(os.environ.get("GITHUB_WORKSPACE")) + "/" + str(os.environ.get("FILE_NAME")), "w") as text_file:
+    text_file.write(out_file)
+text_file.close()
```

Comment:
Do we need this statement? We are using **with** keyword so may be this is not required.
Text from https://docs.python.org/3/tutorial/inputoutput.html
"It is good practice to use the with keyword when dealing with file objects. The advantage is that the file is properly closed after its suite finishes, even if an exception is raised at some point"

Reply:
You are right. Cool if you fix that in the PR :+1: